src/fiat/asset-transfers.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+parent: Fiat
+title: Asset Transfers
+---
+
+# Asset Transfers
+{:.no_toc}
+
+## Contents
+{:.no_toc .text-delta}
+
+* TOC
+{:toc}
+
+## Create an Asset Transfer
+
+{% endpoint POST https://service.centraypay.com/api/asset-transfer-requests %}
+
+```sh
+curl -X POST "https://service.centraypay.com/api/asset-transfer-requests" \
+  -H "x-api-key: 1234"
```

Comment:
Just wondering if this should mention or prefer `Authorization:` headers?

Reply:
We need to add a separate section to explain the different approaches for authenticating API requests. I think it is more natural to use api keys as the default auth method in the docs. We can publish an sdk that makes it easy to make api calls using auth tokens instead. We also need to document our roles and permissions.

Reply:
...but all of this can come later